test/cmdlineTests.sh

```diff
@@ -372,6 +370,41 @@ function test_via_ir_equivalence()
     rm -r "$SOLTMPDIR"
 }
 
+function test_cli_and_standard_json_equivalence
+{
+    (( $# == 5 )) || assertFail
+    local cli_options="$1"
+    local selected_cli_output="$2"
+    local standard_json_settings="$3"
+    local selected_standard_json_output="$4"
+    local input_file_relative_path="$5"
+
+    # CLI normalizes paths, Standard JSON uses them as is. To avoid differences, we want a path
+    # that won't change due to normalization. Only a basic sanity check not covering all cases:
+    [[ "$input_file_relative_path" =~ ^/ ]] && assertfail
+    [[ "$input_file_relative_path" =~ ^\. ]] && assertfail
```

Comment:
Why not?
```suggestion
    [[ "$input_file_relative_path" =~ (^/|\.) ]] && assertfail
```

Reply:
Yeah, that would work. The main reason is that I did no want to spend too much time learning into Bash's odd built-in regex syntax :)